### ISSUE TITLE
code Error in task 1

### ISSUE THREAD

**Issue by presteY** [mdn/content#32859]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Test_your_skills:_Object_basics

### What specific section or headline is this issue about?

code Error

### What information was incorrect, unhelpful, or incomplete?

task1 in this page lose a "dot" in the line "para1.textContent = `The cat's name is ${ cat(HERE)Name }.`;"

### What did you expect to see?

update the live code and the code in the github repository

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/javascript/objects/test_your_skills_colon__object_basics`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Test_your_skills:_Object_basics
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/objects/test_your_skills_colon__object_basics/index.md
* Last commit: https://github.com/mdn/content/commit/c64e813d8ab9dbe22cbc049c26f7c6703370a2b7
* Document last modified: 2023-10-23T08:57:58.000Z

</details>

**Comment by wbamberg** [comment]
Are you saying that this line:

```js
para1.textContent = `The cat's name is ${ catName }.`;
```

...should be:

```js
para1.textContent = `The cat's name is ${ cat.Name }.`;
```

? If so, that's wrong. `catName` is supposed to be a variable that you initialize, as the instructions say:

> Store the value of the name property inside the catName variable, using bracket notation.

**Comment by presteY** [comment]
Yeah, I copied that,  thank you!
________________________________
发件人: wbamberg ***@***.***>
发送时间: 2024年3月29日 3:16
收件人: mdn/content ***@***.***>
抄送: presteY ***@***.***>; Author ***@***.***>
主题: Re: [mdn/content] code Error in task 1 (Issue #32859)


Closed #32859<https://github.com/mdn/content/issues/32859> as not planned.

―
